packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -397,4 +397,33 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
       expect(browser.$$('textarea:focus')).toBeElementsArrayOfSize(1);
     });
   });
+
+  describe('with pinned columns', () => {
+    it('adds pinned columns', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/worklist-data-grid/pinned-columns');
+
+      Terra.validates.element('pinned-columns', { selector });
+    });
+
+    it('keeps pinned columns stickied when scrolling', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/worklist-data-grid/pinned-columns');
+      navigateToCell(2, 1);
+      moveCurrentPositionBy(0, 9);
+      Terra.validates.element('pinned-columns-scroll', { selector });
+    });
+
+    it('keeps row selection column stickied when scrolling', () => {
+      browser.url('/raw/tests/cerner-terra-framework-docs/worklist-data-grid/pinned-columns-with-row-selection');
+      navigateToCell(2, 1);
+      moveCurrentPositionBy(0, 9);
+      Terra.validates.element('pinned-columns-with-row-selection-scroll', { selector });
+    });
+
+    it('selects the whole row with pinned columns', () => {
```

Comment:
I am having a hard time understanding the purpose of this test. It says `selects the whole row` but I do not see that selection on the screenshot ( I only see one cell selected) 
Am I missing something.

Reply:
Good catch, looks like the test file was not setting `hasSelectableRows` correctly. The screenshots should be fixed now.